src/common/config.go

```diff
@@ -14,6 +14,17 @@ import (
 
 // Configuration is stored globally here
 var Conf *Config
+var KafkaConf *KafkaConfig
+
+// KafkaConfig represents the configuration for a Kafka listener
+type KafkaConfig struct {
```

Comment:
We should add support for some per-lambda configs.  Different functions might subscribe to different clusters/topics.